src/routes/auth/index.ts

```diff
@@ -108,7 +108,7 @@ authRoutes.openapi(
 		}
 
 		const userAlreadyActivated = await checkActiveEmail(context.env.Database, email);
-		if (!userAlreadyActivated) {
+		if (userAlreadyActivated) {
 			// INFO: Hide non existing emails to reduce attack surface from guessing registered emails.
```

Comment:
Do we still need this note?

Reply:
I left it there, it is good to have to explain why we bail out of the function earlier